charts/minecraft-proxy/templates/extraports-svc.yaml

```diff
@@ -35,6 +35,8 @@ spec:
   - name: {{ .name }}
     port: {{ .service.port }}
     targetPort: {{ .name }}
+    {{- if .service.nodePort }}
```

Comment:
It looks like this `if` is missing an `end`

There's three more like this.

Reply:
Thanks!  Totally new to if blocks in helm templates.  Fixed!